src/helpers/schema/schema.helper.ts

```diff
@@ -76,47 +76,50 @@ export class SchemaHelper {
   }
 
   public static collect(schema: Schema, objectStack: string[] = [], rootObject?: {}): Schema {
-    if (schema == undefined || schema == null) {
+    if (schema == undefined) {
       return schema;
     }
     if (schema.$ref != undefined) {
-      // let newSchema = SchemaHelper.resolveReference(schema.$ref, rootObject);
-      // for(let key in newSchema){
-      //   schema[key] = newSchema[key];
-      // }
+      if (schema.$ref.includes("com.maxxton")) {
+        let newSchema = SchemaHelper.resolveReference(schema.$ref, rootObject);
+        for (let key in newSchema) {
+          schema[ key ] = newSchema[ key ];
+        }
+      }
+      else console.log("ROBBBBBBBBBBBBBB " + schema.$ref);
```

Comment:
Haha! I can see your name. Can't we have a more meaningful message here 😉

Reply:
Ah busted, a lousy debug message, I'll remove it and change your other comments too. Microdocs code base is quite old, so there's a lot more to improve